.github/CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 <h3>New features since last release</h3>
 
+* Adds a device test suite, located at `pennylane/plugins/tests`, which can be used 
```

Comment:
@mariaschuld ever since you mentioned it, I definitely thing we should rename `pennylane.plugins` to `pennylane.devices`